src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+
+import usersFromServer from '../../api/users';
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. You are importing `usersFromServer` directly in the `TodoForm` component. It's better to import it in the root component (App) and pass it down as a prop to the `TodoForm` component.